packages/wallet/src/wallet.ts

```diff
@@ -191,9 +191,12 @@ export default class Wallet extends AbstractWallet {
     /** Amount of coins */
     amount: BigNumberish,
     /** Asset ID of coins */
-    assetId: BytesLike = NativeAssetId
+    assetId: BytesLike = NativeAssetId,
+    /** Tx Params */
+    txParams: Pick<TransactionRequestLike, 'gasLimit' | 'gasPrice' | 'bytePrice' | 'maturity'> = {}
   ): Promise<TransactionResponse> {
-    const request = new ScriptTransactionRequest({ gasLimit: 1000000 });
+    const params = { gasLimit: 10000, ...txParams };
```

Comment:
Was the change in magnitude of the gasLimit was intentional? 
`1000000` => `10000`

Reply:
@QuinnLee ^